CMakeLists.txt

```
@@ -123,6 +123,11 @@ if(WarpX_PSATD)
     endif()
 endif()
 
+# QED: support for table generation
+if(WarpX_QED_TABLE_GEN)
+    find_package(Boost REQUIRED)
```

```suggestion
    find_package(Boost REQUIRED COMPONENTS math)
```

Although I am not sure if that might better be an option in PICSAR. We can forward this request through `PICSAR.cmake`:
- `set(VARIABLE ...)` in WarpX' code base (see AMReX and openPMD dependencies)
- add table-generation option to upstream PICSAR `CMakeLists.txt` :)